tensorflow/python/ops/math_ops.py

```diff
@@ -4228,6 +4228,12 @@ def polyval(coeffs, x, name=None):
 
      p(x) = coeffs[n-1] + x * (coeffs[n-2] + ... + x * (coeffs[1] +
             x * coeffs[0]))
+            
+  Usage Example:
```

Comment:
Thanks. Please add newline to separate text and code blocks for everything to render on tensorflow.org